packages/ckeditor5-list/src/documentlist/documentlistediting.ts

```diff
@@ -506,12 +511,17 @@ export default class DocumentListEditing extends Plugin {
 		//	                       │  * bar]             │ * bar             │
 		//	                       └─────────────────────┴───────────────────┘
 		//
-		// See https://github.com/ckeditor/ckeditor5/issues/11608.
-		this.listenTo<ModelGetSelectedContentEvent>( model, 'getSelectedContent', ( evt, [ selection ] ) => {
-			const isSingleListItemSelected = isSingleListItem( Array.from( selection.getSelectedBlocks() ) );
+		// See https://github.com/ckeditor/ckeditor5/issues/11608, https://github.com/ckeditor/ckeditor5/issues/14969
+		this.listenTo<ViewDocumentClipboardOutputEvent>( viewDoc, 'clipboardOutput', ( evt, data ) => {
+			if ( data.method != 'dragstart' ) {
+				const selection = model.document.selection;
```

Comment:
I'm not sure if this is safe, in the previous implementation the provided selection was used and it could be different than the current document selection. Maybe we could check the `content` instead?